WcaOnRails/config/application.rb

```diff
@@ -78,6 +78,8 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join('lib')
     config.eager_load_paths << Rails.root.join('lib')
 
+    config.action_mailer.preview_path = "#{Rails.root}/lib/mailer_previews"
```

Comment:
I'm actually out of my knowledge area here: could you explain a bit why this is necessary and what it does?
(ok I've reached the end of the PR and now understand: keeping this comment here to remember that this probably needs to go)

Reply:
This configuration tells rails where to lookup [mail preview]((https://api.rubyonrails.org/classes/ActionMailer/Base.html#class-ActionMailer::Base-label-Previewing+emails) ) classes. For automatic mail preview generation.

Reply:
Oh that does sound awesome, it means we could get rid of mailcatcher at some point, right?

Maybe not in this PR, but how do you feel about improving our documentation by instructing the reader how to preview/what to open to preview emails (taking the reminder emails as an example for instance)?
If my understanding is correct and if you feel really motivated, how do you feel about migrating a couple of our emails to this system so that we can simply say goodbye to mailcatcher?